app/assets/javascripts/pages/Error404.js

```diff
@@ -0,0 +1,77 @@
+const Error404 = {
+    render : async () => {
+        let view =  `
+            <!DOCTYPE html>
+            <html>
+            <head>
+              <title>The page you were looking for doesn't exist (404)</title>
+              <meta name="viewport" content="width=device-width,initial-scale=1">
+              <style>
+              .rails-default-error-page {
+                background-color: #EFEFEF;
+                color: #2E2F30;
+                text-align: center;
+                font-family: arial, sans-serif;
+                margin: 0;
+              }
+
```

Comment:
You should refrain to add css styling in the same same, instead make a separate file for css and then include it

Reply:
will do that.